### ISSUE TITLE
How to test the accuracy of samplot-ml

### ISSUE THREAD

**Issue by AJ2802** [mchowdh200/samplot-ml#5]
Hi, I am validating your trained samplot-ml model using HG002. I ran manta using HG002 WGS short read sequence and genome reference hg38 alt-aware with HLAs provided on Illumina Basespace platform. However, is it appropriate to use this sample and the reference?

The sample concern is mainly because your paper did not remove HG002 from your short read training data.

For the reference concern, the instruction on your samplot-ml git page advise us to use GRCh38_full_analysis_set_plus_decoy_hla.fa. However, it is not provided on Basespace. How much is the accuracy affected if I use the reference genome hg38 alt-aware with HLAs instead?

Thank you.

**Comment by mchowdh200** [comment]
Hi!  We did not use HG002 in the training set of Samplot-ML, so you should be fine.

**Comment by mchowdh200** [comment]
The GRCh38_full_analysis_set_plus_decoy_hla.fa reference can be found in the [1000 genomes ftp](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwjYjPSXueX3AhUXHjQIHWYOCckQFnoECAcQAQ&url=ftp%3A%2F%2Fftp.1000genomes.ebi.ac.uk%2Fvol1%2Fftp%2Ftechnical%2Freference%2FGRCh38_reference_genome%2FGRCh38_full_analysis_set_plus_decoy_hla.fa&usg=AOvVaw1jAWSpLWJ92JpbgDYd93PZ), but as long as you are consistently using the same reference for aligning/calling SVs/generating images, then there shouldn't be an issue.
